core/modules/commands/rendertiddlers.js

```diff
@@ -43,14 +44,19 @@ Command.prototype.execute = function() {
 	if(deleteDirectory){
 		$tw.utils.deleteDirectory(pathname);
 	}
-	$tw.utils.createDirectory(pathname);
 	$tw.utils.each(tiddlers,function(title) {
 		var parser = wiki.parseTiddler(template),
 			widgetNode = wiki.makeWidget(parser,{variables: {currentTiddler: title}});
 		var container = $tw.fakeDocument.createElement("div");
 		widgetNode.render(container,null);
 		var text = type === "text/html" ? container.innerHTML : container.textContent;
-		fs.writeFileSync(path.resolve(pathname,encodeURIComponent(title) + extension),text,"utf8");
+		var export_path = null;
+		if($tw.utils.hop($tw.macros,"tv-get-export-path")) {
+			export_path = $tw.macros["tv-get-export-path"].run.apply(self,[title]);
+		}		
+		var final_path = path.resolve(outputPath,(export_path || encodeURIComponent(title)) + extension);
```

Comment:
Is the problem that here you should be referencing `pathname` instead of `outputPath`?